src/docfx/build/page/BookmarkValidator.cs

```diff
@@ -52,13 +52,14 @@ public void Validate()
             {
                 // #top is HTMl predefined URL, which points to the top of the page
                 if (bookmark == "top")
-                {
                     continue;
-                }
-                if (bookmarksByFile.TryGetValue(reference, out var bookmarks) && bookmarks.Contains(bookmark))
-                {
+
+                // Do not validate bookmark if the target file doesn't report bookmarks
```

Comment:
what's the meaning of "doesn't report bookmakrs"? is it built without any bookmarks or not built at all?